### ISSUE TITLE
[Bug] Its Working But Not Working

### ISSUE THREAD

**Issue by apps-adil** [BiscuitDevelopment/SkyblockAddons#490]
Please note that we do not provide support for clients other than Forge.
We will not fix compatibility issues with other clients.

**Describe the bug**
The mod half works. The mana bar doesn't show and neither the stats. 

**To Reproduce**
I reinstalled and it still doesn't work.. Don't know how to reproduce

**Expected behavior**
Mana, protection, and Magma boss should show

**Screenshots/Video**
If applicable, add screenshots, or a video to help explain your problem.
![image](https://user-images.githubusercontent.com/41599884/105529459-6e570f00-5cb4-11eb-9103-adec9de72737.png)

```text
Does that work? I am using LabyMod as well but that shouldn't be an issue as it was working before.
```
![image](https://user-images.githubusercontent.com/41599884/105530383-b460a280-5cb5-11eb-84d3-02d4a3377cd9.png)

```text
It also seams that it half works as well here, where I can fast travel but doesn't show the bar
```
**System Information**
 - OS: Windows 10
 - Minecraft Version: 1.8.9
 - Forge Version: 1.8.9-11.15.1.1855
 - SkyblockAddons Version: Latest

**Other Installed Mods**
LabyMod

**Comment by MySixSenses** [comment]
Happening with me too

**Comment by MySixSenses** [comment]
I think it's because it somehow can no longer detect you joined Skyblock. Normally the logs or console output something like
[SkyblockAddons]: Joined Skyblock

**Comment by onkel69** [comment]
skyblock switcher some bungeecord stuff 
check #487

**Comment by apps-adil** [comment]
Guess they have to update it so that it works again

**Comment by chaseisprograming** [comment]
Yea same thing with me GUI doesn't show up /sba works haven't tried the fancy warp menu tho.

**Comment by ILikePlayingGames** [comment]
Fixed in v1.5.5
